package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@paddle/paddle-node-sdk",
-  "version": "2.5.1",
+  "version": "2.5.2",
```

```suggestion
  "version": "2.6.0",
```

As we are adding a new feature, we should be incrementing the `minor` version